x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/date/DateParseTests.java

```diff
@@ -103,9 +103,8 @@ public static Iterable<Object[]> parameters() {
         );
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/104124")
     public void testInvalidPattern() {
-        String pattern = randomAlphaOfLength(10);
+        String pattern = "invalid";
```

Comment:
was the randomisation removal intentional?

Reply:
Yes, it was intentional after Ryan's review. The problem is that randomizing the string does not guarantee that you'll get an invalid pattern, so it will make the test flaky. And producing random strings that are always invalid date patterns is not straight-forward